src/Controls/tests/UITests/Tests/Issues/Issue19159.cs

```diff
@@ -0,0 +1,25 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues
+{
+	public class Issue19159 : _IssuesUITest
+	{
+		public Issue19159(TestDevice device) : base(device)
+		{
+		}
+
+		public override string Issue => "Top tab bar on Shell hides content";
+
+		[Test]
+		public void ContentShouldNotBeOverlaidByTopBar()
+		{
+			this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.Mac, TestDevice.Windows | TestDevice.Android });
+			_ = App.WaitForElement("page1");
+
+			// The content should not be overlaid by top bar
+			App.Screenshot();
```

Comment:
Could use here `App.VerifyScreenshot`?

Reply:
Sure!. Should I change it in all my UI tests? I think I wrote `App.Screenshot();` instead of `VerifyScreenshot` in all my PRs 😅

Reply:
It depends. If there are checks validating properties such as text, sizes, etc. is a correct additional validation. But if we are going to validate something (colors, sizes, etc.) with a screenshot, `VerifyScreenshot `is the correct one.

Reply:
Thank you!